src/style.css

```diff
@@ -0,0 +1,132 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,200&family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --color-blue: #00acdc;
+  --color-dark: #060b35;
+  --color-white: #fff;
+  --color-light-grey: #616070;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400px;
+}
+
+.card {
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: var(--color-white);
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--color-dark);
+}
+
+.card__product-code {
+  margin-top: 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-light-grey);
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars {
```

Comment:
Create separate files for card and stars BEM blocks
![image](https://user-images.githubusercontent.com/96409563/227461386-b5fe071a-7101-49a6-bd00-f8fe610a8258.png)

Reply:
still not fixed. please create separate file for each BEM block